docs/postprocessors/alchemlyb.postprocessors.units.rst

```diff
@@ -0,0 +1,23 @@
+﻿alchemlyb.postprocessors.units
+==============================
+
+.. automodule:: alchemlyb.postprocessors.units
+
+.. autofunction:: to_kT
+
+.. autofunction:: to_kcalmol
+
+.. autofunction:: to_kJmol
+
+   
+
```

Comment:
Also document the data and the `get_converter` function. Make a separate section "Constants and auxiliary functions" and put the real converters in a section "Unit converters".